client/text.cpp

```diff
@@ -1712,7 +1713,16 @@ const QString format_duration(int duration)
   } else {
     str += QStringLiteral("%1").arg(Q_("?duration:overflow"));
   }
+  // Show time if there is more than 1hour left
+  if (duration > 3600) {
+    QDateTime time = QDateTime::currentDateTime();
+    QDateTime tc_time = time.addSecs(duration);
+    QString day_now = time.toString("ddd ");
+    QString day_tc = tc_time.toString("ddd ");
 
+    str += QStringLiteral("\n") + ((day_now != day_tc) ? day_tc : "")
+           + tc_time.toString("hh:mm");
```

Comment:
From the docs:

> Note: If localized month and day names are desired, please switch to using QLocale::system().toString() as QDateTime methods shall change to use English (C locale) names at Qt 6.

Do it now to avoid surprises later?